tests/unit/Makefile.am

```diff
@@ -90,7 +90,7 @@ libdb_la_SOURCES += ../../libpromises/cf3globals.c
 endif
 libdb_la_LIBADD = libstr.la ../../libutils/libutils.la
 #libdb_la_CPPFLAGS = $(LMDB_CPPFLAGS) $(TOKYOCABINET_CPPFLAGS) $(QDBM_CPPFLAGS)
-# Make sure that source files are compiled to separate object files 
+# Make sure that source files are compiled to separate object files
```

Comment:
Uh. I have configured vim to remove spaces at the end of lines when saving. Should I exclude this from the PR?

Reply:
Ideally, put changes like this into a separate commit like "whitespace cleanup".

Reply:
"Always" use `git add -p`. Then you are concious about changes like this. You can either skip it, or make it a separate commit.

Reply:
Or `git commit -p`.

Reply:
But definitely not a big deal in this case with just one line changed.